smac/facade/smac_facade.py

```diff
@@ -74,20 +76,29 @@ class SMAC(object):
     def __init__(self,
                  scenario: Scenario,
                  tae_runner: typing.Optional[typing.Union[ExecuteTARun, typing.Callable]]=None,
+                 tae_runner_kwargs : dict={},
```

Comment:
[Here's a reason](https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python) to not use a dictionary as the default.

Reply:
okay, good point